orttraining/orttraining/python/training/onnxblock/checkpoint_utils.py

```diff
@@ -25,14 +25,6 @@ def save_checkpoint(parameters, path_to_checkpoint):
 def load_checkpoint_to_model(path_to_checkpoint, model):
     """Loads the checkpoint to an onnx inference model."""
 
-    # Load the parameters from the checkpoint
-    parameters = _internal_load_checkpoint(path_to_checkpoint)
-
-    parameters_dict = {}
-    for param in parameters:
-        param_proto = TensorProto()
-        param_proto.ParseFromString(param)
-        parameters_dict[param_proto.name] = param_proto
-
-    for initializer in model.graph.initializer:
-        initializer.CopyFrom(parameters_dict[initializer.name])
+    model_proto = ModelProto()
+    model_proto.ParseFromString(_internal_load_checkpoint(path_to_checkpoint, model.SerializeToString()))
+    return model_proto
```

Comment:
Do you think we could make changes in place instead of returning a new ModelProto? `load_checkpoint_to_model` sounds like you would want to load a checkpoint on the given model as opposed to create a new model with the checkpoint?

Reply:
+1

Reply:
You can refer: https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/graph/graph.cc#L3398